exes/mqtt-broadcaster/inc/config.hpp

```diff
@@ -0,0 +1,42 @@
+#pragma once
+
+#include <tfc/confman.hpp>
+#include <tfc/confman/observable.hpp>
+
+// File under /etc/tfc/mqtt-broadcaster/def/mqtt_broadcaster.json which lists the signals that are supposed to be published
+// on the mqtt broker. Updating the signals in this file will cause the running program to stop and restart
+struct config {
+  std::string address{};
```

Comment:
could we transpose asio::endpoint with the config?

Reply:
Could you please explain further? I'm not sure I understand.